gaurun/client.go

```diff
@@ -5,7 +5,7 @@ import (
 	"net/http"
 	"time"
 
-	"github.com/mercari/gcm"
+	"github.com/mercari/gaurun/service/google"
```

Comment:
I don't think original package name(**gcm**) should be changed. (Even if gcm package supports FCM.)